images/installer/Dockerfile.upi.ci

```diff
@@ -2,7 +2,6 @@
 # It builds an image containing binaries like jq, terraform, awscli, oc, etc. to allow bringing up UPI infrastructure.
 # It also contains the `upi` directory that contains various terraform and cloud formation templates that are used to create infrastructure resources.
```

Comment:
nit: we can remove mention of terraform here in the top-level description?

Reply:
done!